CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).
+
+## The beginner's guide to contributing to a GitHub project
+
+<img align="right" width="300" src="assets/images/fork.png" alt="fork this repository" />
+
+## Fork this repository
+
+Fork this repository by clicking on the fork button on the top of this page.
+This will create a copy of this repository in your account.
+
+## Clone the repository
+
+<img align="right" width="300" src="assets/images/clone.png" alt="clone this repository" />
```

Comment:
I guess the PNG images make this Markdown less mobile friendly, specially since they have fixed width. Could we remove them?

Reply:
Maybe add hyperlinks to the respective GitHub documentation page